src/contracts/atlas/Storage.sol

```diff
@@ -18,6 +18,8 @@ contract Storage is AtlasEvents, AtlasErrors, AtlasConstants {
     address public immutable SIMULATOR;
     address public immutable L2_GAS_CALCULATOR;
     uint256 public immutable ESCROW_DURATION;
+    uint256 public immutable ATLAS_SURCHARGE_RATE;
```

Comment:
are 
`ATLAS_SURCHARGE_RATE`
and 
`keccak256("ATLAS_SOLVER_SURCHARGE");`

The same maybe good to use the same string

Reply:
These are 2 different variables.

- `ATLAS_SURCHARGE_RATE ` is the percentage of gas cost we take as an Atlas surcharge, stored as an immutable variable.
- `solverSurcharge` is a transient variable, used to track the total surcharge (Atlas + Bundler) of the cost of failed solverOp gas costs, during the current metacall, if failure is the solver's fault. This has also been changed in #442 to use the latest `transient` keyword to create the `t_solverSurcharge` instead of how it is done here, where we hash "ATLAS_SOLVER_SURCHARGE" to target a specific transient storage slot.